### PR TITLE
docs: add default props to documentation

### PR DESCRIPTION
**Pull request by HQFOX**

### CODE REVIEW

File: docs/guides/theming/WhiteLabeling.stories.mdx [hunk 1]
```diff
@@ -64,12 +103,12 @@ const MyApp = ({ children }) => {
 };
 ```
 
-Only by customizing the theme, the `HvHeader`, `HvButton`, `HvHeaderBrand`, `HvHeaderNavigation`, `HvGlobalActions`, `HvTypography`, and `HvContainer`
-components were able change their default styling.
+By customizing the theme variables, the `HvHeader`, `HvButton`, `HvHeaderBrand`, `HvHeaderNavigation`, `HvHeaderMenuIte` and `HvHeaderMenuBarBar`
```
Comment (MEsteves22) [inline docs/guides/theming/WhiteLabeling.stories.mdx:106]: I believe there are some typos here, maybe:

"By customizing the theme variables,  we are able to change the default styling of the `HvHeader`, `HvButton`, `HvHeaderBrand`, `HvHeaderNavigation`, `HvHeaderMenuItem`, and `HvHeaderMenuBarBar` components to meet the desired UI guidelines."

File: docs/guides/theming/ThemeStructure.tsx [hunk 2]
```diff
@@ -115,6 +116,34 @@ export const ThemeStructure = () => {
...
+          label: styles.accordionLabel,
+        }}
+        headingLevel={1}
+        label={label}
+      >
+        <HvCodeEditor
+          height={420}
+          language="json"
+          value={JSON.stringify(value, null, "\t")}
+        />
```
Comment (MEsteves22) [inline docs/guides/theming/ThemeStructure.tsx:142]: I believe it would be better to make the editor read-only with `options={{ readOnly: true }}`.

**Review by MEsteves22 — COMMENTED** [review]
Really nice 👌🏻

I just have some minor comments:

File: docs/guides/theming/Theming.stories.mdx [hunk 3]
```diff
@@ -363,6 +365,67 @@ const MyComponent = () => {
...
+<Story name="Default Props">
+  <DefaultProps />
+</Story>
+
+This allows us to not have to declare that property every time we use the component.
+One of the big benefits of this is that most of all of our components have a `classes` prop. This means that we can now customize all
+of the components by overwriting their classes through the theme object.
+
+```tsx
+const newThemeClassOverwrite = createTheme({
```
Comment (MEsteves22) [inline docs/guides/theming/Theming.stories.mdx:402]: It should be `newTheme` and not `newThemeClassOverwrite` as defined in the `HvProvider` below.

Also, could we add the imports just for the sample to be similar to the other ones (and also users may find the imports useful)?

```
import { createTheme, HvProvider, HvButton } from "@hitachivantara/uikit-react-core";
```

File: docs/guides/theming/Theming.stories.mdx [hunk 4]
```diff
@@ -363,6 +365,67 @@ const MyComponent = () => {
   <CssVariables />
 </Story>
 
+## Default Props <a id="default-props" />
+
+Through our theme it is also possible to change the default props of a component. Let's say for example that the theme you want only uses squared
+shape avatars, you can use this feature to make your life easier by pre defining the default value of the prop `variant` to `square` in the `theme` `components` section like so:
+
+```tsx
+const newTheme = createTheme({
```
Comment (MEsteves22) [inline docs/guides/theming/Theming.stories.mdx:374]: Could we add the imports just for the sample to be similar to the other ones (and also users may find the imports useful)?

```
import { createTheme, HvProvider, HvAvatar } from "@hitachivantara/uikit-react-core";
```

**Review by MEsteves22 — APPROVED** [review]

**Review by zettca — APPROVED** [review]
👌

**Pull request merged**